src/ert/callbacks.py

```diff
@@ -44,8 +51,14 @@ def _write_responses_to_storage(
             if not config.keys:
                 continue
         try:
+            start_time = time.perf_counter()
+            logger.info(f"Starting to load response: {config.name}")
             ds = config.read_from_file(run_arg.runpath, run_arg.iens)
             run_arg.ensemble_storage.save_response(config.name, ds, run_arg.iens)
+            logger.info(
+                f"Saved {config.name} to storage in "
+                f"{(time.perf_counter() - start_time):.4f}s"
+            )
```

Comment:
We should also log the timing information as an extra field, such that it is easier to use it later processing.